client-java/controller/src/main/java/org/evomaster/client/java/controller/db/SqlScriptRunner.java

```diff
@@ -359,6 +359,22 @@ public static void execScript(Connection conn, String script) throws SQLExceptio
         }
     }
 
+    /**
+     *
+     * @param conn a connection to db
+     * @param script represents a sql script
+     * @param tablesToInsert represents if only insert table which exists in this list
```

Comment:
i don't understand this comment... need to better clarify it

Reply:
do u mean something like `insertions are executed only on the tables in this list. insertions for other tables will be skipped. `?

Reply:
also, what should happen if `tablesToInsert` is null or empty? should throw an IllegalArgumentException?

Reply:
thanks. doc is updated. 
if it is `null` or `empty`, nothing will be inserted now. This function is mainly used in `resetDatabase` in generated tests.